src/BBCodeParser.php

```diff
@@ -24,7 +24,7 @@ class BBCodeParser extends TextParser {
 	 * @config
 	 * @var Boolean Set whether similies :), :(, :P are converted to images
 	 */
-	private static $allow_similies = false;
+	private static $allow_smilies = false;
```

Comment:
If you meant the PHPDoc, that's been fixed, beyond that I searched the module globally for `simili*` and found nothing